source/_integrations/vesync.markdown

```
@@ -111,6 +114,17 @@ All VeSync air purifiers expose the remaining filter lifetime, and some also exp
 | `air_quality`           | The current air quality reading. (LV-PUR131S, Core300s/400s/600s)                      | excellent |
 | `pm2_5`                 | The current air quality reading. (Core300s/400s/600s/EverestAir)                       | 8         |
 
+## Humidifier
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)